### PR TITLE
brew: add Spanish translation

### PR DESCRIPTION
**Pull request by moisesnandres**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/osx/brew.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# brew
+
+> Administrador de paquetes para macOS.
+> Más información: <https://brew.sh>.
+
+- Buscar fórmulas y paquetes disponibles:
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:6]: Se debe usar el modo indicativo, no infinitivo:
```suggestion
- Busca fórmulas y paquetes disponibles:
```

File: pages.es/osx/brew.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew search {{text}}`
+
+- Instalar la última version estable de una fórmula (usar `--devel` para versiones de desarrollo):
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:10]:
```suggestion
- Instala la última version estable de una fórmula (use `--devel` para versiones de desarrollo):
```

File: pages.es/osx/brew.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew install {{formula}}`
+
+- Mostrar todas las fórmulas instaladas:
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:14]:
```suggestion
- Muestra todas las fórmulas instaladas:
```

File: pages.es/osx/brew.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew list`
+
+- Mostrar fórmulas instaladas que no dependen de otras:
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:18]:
```suggestion
- Muestra fórmulas instaladas que no dependen de otras:
```

File: pages.es/osx/brew.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew leaves`
+
+- Actualizar una fórmula instalada (si no se añade el nombre, se actualizarán todas las fórmulas):
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:22]: This text is not cool in English for me.
Creo que `indicar` es el verbo más semántico para esto:
```suggestion
- Actualiza una fórmula instalada (si no se indica el nombre, se actualizan todas las fórmulas):
```

File: pages.es/osx/brew.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew upgrade {{formula}}`
+
+- Actualizar a la última versión de Homebre y todas sus fórmulas desde GitHub:
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:26]:
```suggestion
- Actualiza a la última versión de Homebrew y todas sus fórmulas desde GitHub:
```

File: pages.es/osx/brew.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew update`
+
+- Remover versiones antiguas de fórmulas instaladas (si no se añade el nombre, se buscará en todas las fórmulas):
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:30]:
```suggestion
- Elimina versiones antiguas de fórmulas instaladas (si no se indica el nombre, se busca en todas las fórmulas):
```

File: pages.es/osx/brew.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew cleanup {{formula}}`
+
+- Mostrar información acerca de una fórmula (versión, ruta de instalación, dependencias, etc.):
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:34]:
```suggestion
- Muestra información acerca de una fórmula (versión, ruta de instalación, dependencias, etc.):
```

File: pages.es/osx/brew.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+`brew info {{formula}}`
+
+- Revisar la instalación local de Homebrew para buscar problemas poteniales:
```
Comment (navarroaxel) [inline pages.es/osx/brew.md:38]:
```suggestion
- Verifica la instalación local de Homebrew en busca de problemas poteniales:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @moisesnandres, welcome and thanks for your submission. I left some comments.

**Comment by moisesnandres** [comment]
Hi @navarroaxel , thanks for the review, I've just updated the file

**Review by navarroaxel — APPROVED** [review]
Thanks @moisesnandres, please **don't** push force, it's harder to use to track the suggestions and changes.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

Feel free to open 1 big PR if you're translating lots of pages, with 1 commit per page  :-)

**Pull request merged**